notebooks/SpectrumListExample.ipynb

```diff
@@ -0,0 +1,114 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "96971b54",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import glob\n",
+    "import tempfile\n",
+    "from zipfile import ZipFile\n",
+    "\n",
+    "from astropy.utils.data import download_file\n",
+    "\n",
+    "from specutils import SpectrumList"
```

Comment:
I don't see any viz used here. Should this really be in a notebook? This reads more like a unit test.

Reply:
Talking about tests, can you add some? Preferably with the smallest working dataset you can find.

Reply:
There are tests in PR #843

Reply:
I am still not convinced this should be in upper level notebook, unless you can tie its usage to one of the viz. But if you move to concepts folder, I have no objection.

Reply:
You'll be able to load a `SpectrumList` directly into Specviz once #843 is merged. Might be worth adding that here?

Reply:
Is this proposal below too controversial?

1. Rebase (not git pull) this PR to get rid of the merge commits.
2. Squash all commits here into one.
3. `git cherry-pick` the result hash here into #843 . The viz integration can continue there.
4. Close this PR *without* merge.

Reply:
Though after posting the above, I saw that Ivo moved it to concept, so 🤷

Reply:
Also, @ibusko , you forgot to clear the notebook outputs.